npm/README.md

```diff
@@ -0,0 +1,114 @@
+# electron-prebuilt
+
+[![Travis build status](http://img.shields.io/travis/electron-userland/electron-prebuilt.svg?style=flat)](http://travis-ci.org/electron-userland/electron-prebuilt)
```

Comment:
These badges won't be accurate any more. Should we point them at electron/electron? Take them out?